pkg/users/local.js

```diff
@@ -750,7 +752,7 @@ PageAccount.prototype = {
            if (data[data.length - 1].indexOf('**Never logged in**') > -1)
                return null;
            else
-               return new Date(data[data.length - 1]);
+               return moment(new Date(data[data.length - 1])).format('LLL');
```

Comment:
Same thing here.
22. März 2018 18:12

Reply:
Ditto

Reply:
Since this parses a string from `last`, can we rely on this always working? Might this require a fallback to show it verbatim if moment is unable to parse it?

Reply:
You're right, I'll add the `isValid()` check here too.